app/middlewares/authCheck.js

```diff
@@ -1,15 +1,21 @@
 const errors = require('../errors'),
   User = require('../models').user,
+  time = require('time'),
   sessionsManager = require('../services/sessionsManager.js');
 
+const MILISECONDS_PER_MINUTE = 60000;
+
 exports.checkUser = (req, res, next) => {
   const auth = req.headers.authorization; // auth is in base64(username:password)  so we need to decode the base64
   try {
     const decoded = sessionsManager.decode(auth);
+    const currentTime = new Date();
+    const minutesSinceLogin = Math.floor((currentTime.getTime() - decoded.exp.time) / MILISECONDS_PER_MINUTE);
+    if (minutesSinceLogin > 1) throw errors.expiredSession;
```

Comment:
here you should call next instead of throwing it, and in the below catch you throw the unauthorised error.